test/integration/servers/graphql-main-server/auth-requests.spec.ts

```diff
@@ -28,28 +25,26 @@ import {
   lndOutside2,
 } from "test/helpers"
 
-jest.mock("@services/twilio", () => require("test/mocks/twilio"))
-
-let apolloServer, httpServer, httpTerminator, query, mutate, setOptions, walletId
-const { phone, code } = yamlConfig.test_accounts[0]
+let apolloClient, disposeClient, walletId
```

Comment:
could you type `walletId: WalletId`?

not sure if the 2 other variables are easily typables but that would be great to have that as well (not a requirement, but we're trying to move to a place where we can have no-implicit-any set to true)

Reply:
I can type them!